packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -14,9 +16,30 @@ import { makePromiseKit } from '@agoric/promise-kit';
 import makeObservablePurse from './observable';
 import { makeDehydrator } from './lib-dehydrate';
 
+import '@agoric/zoe/exported';
+import './types';
+
 // does nothing
 const noActionStateChangeHandler = _newState => {};
 
+const cmp = (a, b) => {
```

Comment:
Validate that this behaves as expected (ascending sort of a list of integers).  We might need to reverse its sense, and instead use `-cmp(a, b)` where we currently have `cmp(a, b)`.